__tests__/app.test.js

```diff
@@ -131,6 +131,67 @@ describe("/api/articles/:article_id", () => {
         expect(response.body.msg).toBe("Bad request");
       });
   });
+  test("PATCH 200: responds with object representing updated article", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 100 })
+      .expect(200)
+      .then(({ body }) => {
+        const { updated_article } = body;
+        expect(updated_article).toMatchObject([
+          {
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 200,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          },
+        ]);
+      });
+  });
+  test("PATCH 400: responds with error when invalid inc_votes value is given for patch request", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: "hello" })
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("Bad request");
+      });
+  });
+  test("PATCH 400: responds with error when invalid object is given for patch request", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({
+        username: "icellusedkars",
+        body: "wow, this article is stupendous!",
```

Comment:
For this one, I'm actually going to make a suggestion. In order for your API to be _open to expansion_ **without** breaking this test, do we want unnecessary properties to give an error, or be ignored?
What if later on you decide you want a user to be able to patch the body? 🤔

Reply:
Good point Mitch! I have now added a test to ensure that the patch ignores additional properties, as long as an inc_votes property is included in the request object!